docs/dev_guide/ceph_s3_tests/ceph_s3_tests_guide.md

```diff
@@ -287,4 +288,46 @@ S3TEST_CONF=${PWD}/src/test/system_tests/ceph_s3_tests/test_ceph_s3_config.conf
 
 Notice that even though test commands succeeded the test itself was skipped. The test prints `1 skipped` meaning one test was skipped
 
+## Run a Single Ceph S3 Test in NC deployment Manually
+
+### NC local on your machine
+#### Account creation
+Create the needed accounts for the Ceph configurations, for example: 
+1. ceph (main account)
+2. ceph_alt (alternative account - for example I saw it used in one of the bucket policy tests)
```

Comment:
just writing "alternative account" may be confusing. maybe its worth mentioning that it is used on tests when two different accounts are needed

Reply:
Added, I hope it is clearer now.